openapi/src/test/resources/petstore.yaml

```diff
@@ -37,7 +37,7 @@ paths:
             type: integer
             format: int32
       responses:
-        200:
+        '200':
```

Comment:
Is this required? This seems like quite a big backward incompatible change - if somebody uses numbers in current version, will this fail?

Reply:
A document which excludes the quotes around the HTTP status codes for response is in violation of the OpenAPI spec: https://swagger.io/specification/#responses-object:
> HTTP status code field: ...This field MUST be enclosed in quotation marks (for example, "200") for compatibility between JSON and YAML.

Our test file should never have omitted the quotation marks. That said, in the past we would accept this error in documents. I am looking at how complicated it would be to customize the parsing to continue to accept this type of error (and log warnings as we do so).

Reply:
Flagging these with warnings while accepting them is relatively simple. Another push will arrive shortly.